### PR TITLE
docs: Fix broken links

### LINKED ISSUE #20: Dead links in packet docs
Links in the modules table are dead https://github.com/kinvolk/lokomotive-kubernetes/blob/master/docs/index.md#modules

### PR DESCRIPTION
**Pull request by surajssd**
Fixes https://github.com/kinvolk/lokomotive-kubernetes/issues/20

### CODE REVIEW

File: docs/advanced/customization.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Lokomotive AWS, Azure, and bare-metal support CLC *snippets* - valid Container L
 
 #### Examples
 
-Container Linux [docs](https://coreos.com/os/docs/latest/clc-examples.html) show many simple config examples. Ensure a file `/opt/hello` is created with permissions 0644. 
+Container Linux [docs](https://coreos.com/os/docs/latest/clc-examples.html) show many simple config examples. Ensure a file `/opt/hello` is created with permissions 0644.
```
Comment (invidian) [inline docs/advanced/customization.md]: This is not fixing broken links. Cleanups should go to another commit.

**Review by invidian — APPROVED** [review]
Looks fine except some nits.

**Comment by surajssd** [comment]
Reverted back to what this PR's original intent was!

**Review by invidian — APPROVED** [review]
